cmd/testscript/playgroundclient/client.go

```diff
@@ -0,0 +1,87 @@
+package playgroundclient
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+)
+
+type Config struct {
+	BaseURL string
+}
+
+type PlaygroundClient struct {
+	baseURL string
+	client  *http.Client
+}
+
+type PostRunsRequest struct {
+	Database    string `json:"database"`
+	Version     string `json:"version"`
+	Query       string `json:"query"`
+	SaveRunInfo bool   `json:"save_run_info"`
+}
+
+type PostRunsResponse struct {
+	Result PostRunsResult `json:"result"`
+}
+
+type PostRunsResult struct {
+	TimeElapsed string `json:"time_elapsed"`
+}
+
+func New(c *Config) *PlaygroundClient {
+	return &PlaygroundClient{
+		baseURL: c.BaseURL,
+		client:  &http.Client{Timeout: 0},
+	}
+}
+
+func (c *PlaygroundClient) PostRuns(database string, version string, query string) (time.Duration, error) {
+	url := c.baseURL + "/api/runs"
+
+	requestBody := PostRunsRequest{
```

Comment:
I see two ways to organize this tool sources:

1. Testscript source code is fully isolated. The `/cmd/testscript` directory can be copied and it will work. In that case, the tool should have its own go.mod.
2. We share go.mod and all the dependencies, and refer to the main playground packages in this tool. In that case, we need to move all the package from the `cmd` directory to the `internal` and `pkg` dirs. Also, we can refer to the `PostRunsRequest` structure from the API package.

The second strategy reminds me about monorepository. We can go with to have better dependency control.